packages/framework/src/Resources/translations/messages.cs.po

```diff
@@ -949,6 +967,15 @@ msgstr "Při nahrávání souboru došlo k chybě: %message%"
 msgid "Error occurred, try again please."
 msgstr "Nastala chyba, zkuste to, prosím, znovu."
 
+msgid "Every 5 minutes in %hour% hour"
+msgstr "[0,1]Každých 5 minut v %hour% hodinu|[2,4]Každých 5 minut ve %hour% hodiny|[2,Inf]Každých 5 minut v %hour% hodin"
```

Comment:
It is weird. You have three intervals [0,1], [2,4] and [2,-], First interval will return `Každých 5 minut v 0 hodin-u-` and
`Každých 5 minut v 1 hodinu`, second interval is ok, third interval should be [5, Inf], or not?